src/pages/lib/logging/hub-events/q/platform/[platform].mdx

```diff
@@ -0,0 +1,15 @@
+export const meta = {
+  title: `Logging Events`,
```

Comment:
can we please change the title of the page to "Listening to logging events"

Reply:
Title updated.